src/models/configentry.py

```diff
@@ -0,0 +1,9 @@
+from sqlmodel import Field, SQLModel
+from typing import Union
+
+
+class ConfigEntry(SQLModel, table=True):
+    id: Union[int, None] = Field(default=None, primary_key=True)
```

Comment:
```suggestion
    id: int | None = Field(default=None, primary_key=True)
```

It'd be cool to use the new typing hints if our python is new enough

Reply:
It's not yet (but we'll get there)